src/transformers/models/bit/image_processing_bit.py

```diff
@@ -79,10 +79,11 @@ class BitImageProcessor(BaseImageProcessor):
             Mean to use if normalizing the image. This is a float or list of floats the length of the number of
             channels in the image. Can be overridden by the `image_mean` parameter in the `preprocess` method.
         image_std (`float` or `List[float]`, *optional*, defaults to `OPENAI_CLIP_MEAN`):
-            Image standard deviation.
-        do_convert_rgb (`bool`, *optional*, defaults to `True`):
             Standard deviation to use if normalizing the image. This is a float or list of floats the length of the
             number of channels in the image. Can be overridden by the `image_std` parameter in the `preprocess` method.
+            Can be overridden by the `image_std` parameter in the `preprocess` method.
```

Comment:
```suggestion
            Can be overwritten by the `image_std` parameter in the `preprocess` method.
```

Reply:
I think both are fine, but `be overridden` has > 100 occurrences while `be overwritten` has only 22 occurrences. I will keep `overridden` here.